numba_cuda/numba/cuda/np/npyfuncs.py

```diff
@@ -51,7 +51,7 @@ def _check_arity_and_homogeneity(sig, args, arity, return_type=None):
         assert False, msg
 
 
-if config.USE_LEGACY_TYPE_SYSTEM:
+if getattr(config, "USE_LEGACY_TYPE_SYSTEM", True):
```

Comment:
For future reference, you can always assume `USE_LEGACY_TYPE_SYSTEM` is `True`, and not include the code for the case when it is `False`.